sync_diff_inspector/main.go

```diff
@@ -104,3 +106,12 @@ func checkSyncState(ctx context.Context, cfg *config.Config) bool {
 	}
 	return d.PrintSummary(ctx, &cfg.Task)
 }
+
+func startPProf(cfg *config.Config) error {
+	// Initialize the pprof server.
+	if cfg.StatusAddr != "" {
+		return sync_utils.StartPProfListener(cfg.StatusAddr)
+	}
+	sync_utils.StartDynamicPProfListener()
```

Comment:
I perfer to provide a default status-addr instead of dynamic address

Reply:
default status-addr may conflict...